Services/BrokerService/Controllers/BrokerController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using BrokerService.Interfaces;
+using BrokerService.Utils;
+using DTO;
+using DTO.MarketBrokerObjects;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace BrokerService.Controllers
+{
+    [Route("[controller]")]
+    public class BrokerController : Controller
```

Comment:
If i'm not mistaken, this service is not a rest one

Reply:
+